gulp-core-build/src/tasks/CleanTask.ts

```diff
@@ -59,8 +58,17 @@ export class CleanTask extends GulpTask<void> {
       }
     }
 
-    del(cleanPaths)
-      .then(() => completeCallback())
-      .catch((error) => completeCallback(error));
+    return Promise.all(
+      cleanPaths.map((cleanPath) => {
+        return new Promise<void>((complete: () => void, reject: (err: Error) => void) => {
```

Comment:
This code might be more readable if you created a standalone function to adapt rimraf() to return a promise.